app/src/datasets/odrive/file_conversion.py

```diff
@@ -0,0 +1,43 @@
+import os
+import pandas as pd
+import math
+import datetime
+
+names_to_replace = {
+    "CLIO IV": "CLIO 4",
+    "CLIO III": "CLIO 3",
+    "MÉGANE": "MEGANE",
+    "ZOÉ": "ZOE",
+}
+
+
+def clean_modele(modele):
+    modele = str(modele).upper()
+    if modele in names_to_replace.keys():
+        modele = names_to_replace.get(modele)
+    return modele
+
+
+def clean_date(date_value):
```

Comment:
How much of the dataset needs this operation to be performed?
Shouldn't we drop rows without a correct date?

Reply:
192 dates are "invalid" here, The mileage is still provided for the vehicle so I prefer to replace the dates and "dilute" the kilometers over the years instead of losing the vehicle/emissions.
But we need to keep in mind that information to avoid any skew later.

Reply:
Ok.